### PR TITLE
Codechange: Make codestyle for CMake files consistent for 'control' statements

### PR DESCRIPTION
**Pull request by LordAro**
Mostly, remove the confusing and largely redundant text from end*/else statements.

Feel free to argue about whether there should be a space or not.

```
sed -i 's/\(if\|macro\|elseif\|function\|foreach\)\s*(\(.*\))/\1 (\2)/' cmake/*.cmake CMakeLists.txt
sed -i 's/\(endif\|endmacro\|else\|endfunction\|endforeach\)\s*(.*)/\1 ()/' cmake/*.cmake CMakeLists.txt
```

### CODE REVIEW

**Comment by glx22** [comment]
I'd say no space

**Comment by FLHerne** [comment]
+1 for consistency.

The [official documentation](https://cmake.org/cmake/help/latest/command/if.html) and other CMake projects I've seen omit the space -- in CMake syntax, `if()` etc. are ordinary commands and not special keywords.

As I said on IRC, I don't think that matters so long as it's internally consistent.

**Comment by LordAro** [comment]
space removed

File: cmake/FindFontconfig.cmake [hunk 1]
```diff
@@ -19,7 +19,7 @@ Result Variables
 This will define the following variables in your project:
 
 ``Fontconfig_FOUND``
-  true if (the requested version of) Fontconfig is available.
+  true if(the requested version of) Fontconfig is available.
```
Comment (glx22) [inline cmake/FindFontconfig.cmake]: ah `sed` was too aggressive :)

Comment (LordAro) [reply cmake/FindFontconfig.cmake]: ahaha

**Comment by LordAro** [comment]
Found all the other cmake files as well...

```
find  \( -name '*.cmake' -o -name 'CMakeLists.txt' \) -not -path './build/*' -exec sed -i 's/\(if\|macro\|elseif\|function\|foreach\)\s*(\(.*\))/\1(\2)/' {} \; -exec sed -i 's/\(endif\|endmacro\|else\|endfunction\|endforeach\)\s*(.*)/\1()/' {} \;
```

**Review by glx22 — APPROVED** [review]
seems good

**Pull request merged**